index.html

```diff
@@ -6,11 +6,211 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css" />
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <section class="homepage-container">
+        <section class="hero-box">
+                <section class="hero-flexbox">
```

Comment:
What happened to using `<header>, `<main>` and `<footer>`? Please edit your HTML to use a semantic structure. 
Having `section`s inside `main` is good to split the main part of the page into logical blocks, but these must all live inside the `main` tag